modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -125,6 +128,34 @@ object WebServerLauncher extends ProcessApp with LogInitialization {
     Task.delay { logger.info(msg) }
   }
 
+  // We need to manually update the configuration of the logging subsystem
+  // to support capturing log messages and forward them to the clients
+  def logToClients(out: Topic[SeqexecEvent]): Task[Appender[ILoggingEvent]] = Task.delay {
+    import org.slf4j.LoggerFactory
+    import ch.qos.logback.classic.LoggerContext
+    import ch.qos.logback.classic.Logger
+    import ch.qos.logback.classic.AsyncAppender
+
+    val asyncAppender = new AsyncAppender
+    val appender = new AppenderForClients(out)
```

Comment:
Why 2 loggers appenders now?

Reply:
After your comment it got me thinking that we could be blocking on a thread. The `AsyncAppender` is just a wrapper that will keep events in a queue and send them in another thread avoiding this problem.